backends/ebpf/psa/README.md

```diff
@@ -141,7 +141,17 @@ An inner map is a hash map storing clone session/multicast group members, accord
 `struct list_key_t` (BPF map key) and `struct element` (value). To add a new clone session/mutlicast group, a control plane must add
 a new element to the outer map (indexed by clone session or multicast group identifier referenced by `clone_session_id` 
 or `multicast_group` in a PSA program) and initialize an inner map. To add a new clone session/multicast group member,
-a control plane must add new element to the inner map.
+a con1trol plane must add new element to the inner map.
+
+## PSA externs
+
+### Digest
+
+Digests are intended to carry a small piece of user-defined data from the data plane to a control plane.
+The PSA-eBPF compiler translates each Digest instance into `BPF_MAP_TYPE_QUEUE` that implements a FIFO queue.
```

Comment:
I guess you have a separate table to keep them strongly typed, instead of one table with a union?
This will lose the relative ordering of digests between different tables.

Reply:
For a single Digest instance, there is a separate BPF map of type `BPF_MAP_TYPE_QUEUE`. No unions, etc. Just a straightforward mapping.

Reply:
There is no requirement in PSA that data plane -> controller ordering must be maintained across different instances of Digest externs.